Resources/Storage/StorageFactory.py

```diff
@@ -253,13 +260,13 @@ def _getConfigStorageProtocolDetails( self, storageName, protocolSection ):
       configPath = cfgPath( protocolConfigPath, option )
       optionValue = gConfig.getValue( configPath, '' )
       protocolDict[option] = optionValue
-        
+
     # This is a temporary for backward compatibility
-    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:  
+    if "ProtocolName" in protocolDict and not protocolDict['PluginName']:
       protocolDict['PluginName'] = protocolDict['ProtocolName']
-    protocolDict.pop( 'ProtocolName' )  
-        
-    # Evaluate the base path taking into account possible VO specific setting 
+    protocolDict.pop( 'ProtocolName' )
```

Comment:
Either this has to go inside the "if" statement, or in a try catch, or pop('ProtocolName', None)